charts/keycloakx/templates/statefulset.yaml

```diff
@@ -145,6 +145,9 @@ spec:
             - name: http
               containerPort: 8080
               protocol: TCP
+            - name: http-internal
```

Comment:
```suggestion
            - name: '{{ .Values.http.internalPort }}'
```

Reply:
Fixed it

Reply:
Anything else missing?

Reply:
Seems lik I missed removing a deprecated argument, this the --hostname-strict-https argument. It should be working now.

Reply:
After testing, I also discovered the deprecated proxy option. So I updated the values from the upgrade documentation here:
- https://www.keycloak.org/docs/latest/upgrading/index.html#deprecated-proxy-option

I think it will be too difficult to try a full backwards compatibility version for this one. If someone wants to run an older version of Keycloak, then I would advice turning the proxy of in the values file and adding necessary env variables manually.

Reply:
Looks good. Just sign off your commit and I will merge it and figure out how to release this 😅 
https://github.com/codecentric/helm-charts/pull/770/checks?check_run_id=26551409269

Reply:
Signed and done